RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -219,9 +219,9 @@ boolean auto_pre_adventure()
 		}
 	}
 
-	// Use some instakills.
+	// Use some instakills.  Removing option from Pocket Familiars so it won't unnecessarily equip in third slot.
 	item DOCTOR_BAG = $item[Lil\' Doctor&trade; Bag];
-	if(auto_is_valid(DOCTOR_BAG) && possessEquipment(DOCTOR_BAG) && (get_property("_chestXRayUsed").to_int() < 3) && my_adventures() <= 19)
+	if(auto_is_valid(DOCTOR_BAG) && possessEquipment(DOCTOR_BAG) && (get_property("_chestXRayUsed").to_int() < 3) && my_adventures() <= 19 && !in_pokefam())
```

Comment:
see fantasy bandits comment below.

Reply:
Same comment about fantasyrealm not available in pokefam